ui/src/main.ts

```diff
@@ -5,7 +5,7 @@ import store from './store'
 import VueDiff from 'vue-diff'
 
 import '@featherds/styles'
-import '@featherds/styles/themes/open-light.css'
+import '@featherds/styles/themes/open-light.scss'
```

Comment:
I think we should keep the .css extension here. I know it was originally changed because of a feather update, but that was a bug which was fixed in feather v10.02